src/components/UI/Article/Article.jsx

```diff
@@ -16,46 +16,60 @@ const article = (props) => {
         slug,
         loading,
         image,
-        body
+        body,
+        facebook,
+        twitter,
+        email
     } = props;
     const renderedBody = renderHTML(body)
     const defautImage = 'https://images.pexels.com/photos/1591057/pexels-photo-1591057.jpeg?cs=srgb&dl=buttons-characters-concept-1591057.jpg&fm=jpg';
+    const isLoggedIn = localStorage.getItem('token');
 
-    return ( 
+    return (
         <div>
             <Container>
                 <Header as='h1' padded="very">
                     {title}
                 </Header>
-            </Container> 
+            </Container>
             <Container>
                 {author}
                 <Container>
-                    {dateCreated} <p>{readTime}</p>
+                    {dateCreated}
+                    <p>{readTime}</p>
                 </Container>
             </Container>
-            { isAuthor ? 
+            {isAuthor ? (
                 <div className="actionButtons">
                     <Container textAlign='right'>
                         <Button
                             color="red"
                             basic
-                            onClick={ deleteArticle }
-                            >Delete</Button>
-                            <Link className="article-card-title" to={`/articles/update/${slug}`}>
-                                <Button color="teal" basic loading={loading}>Update</Button>
-                            </Link>
+                            onClick={deleteArticle}>
+                            Delete
+                        </Button>
+                        <Link className="article-card-title" to={`/articles/update/${slug}`}>
+                            <Button color="teal" basic loading={loading}>Update</Button>
+                        </Link>
                     </Container>
-                </div> : null 
+                </div>
+            ) : null
             }
             <Container textAlign='justified'>
-                <Image src= { image ? image : defautImage} />
+                <Image src={image ? image : defautImage} />
                 <div className="ArticleBody">
                     {renderedBody}
                 </div>
             </Container>
+            {isLoggedIn ? (
+                <Container textAlign="center">
+                    <a href={facebook} style={{ color: "#4267b2" }}><Icon name="facebook f" style={{ fontSize: "24px" }} /></a>
```

Comment:
Is there any reason we are the `styling` is here and not in the `css` files?

Reply:
Just worked on this